guides/common/modules/proc_adding-azure-connection.adoc

```diff
@@ -1,7 +1,40 @@
 [id="adding-azure-connection_{context}"]
 = Adding a Microsoft Azure Resource Manager Connection to {ProjectServer}
 
-Use this procedure to add Microsoft Azure Resource Manager as a compute resource in {Project}.
+To obtain the credentials needed from Microsoft Azure, follow the below procedure:
+
+.Procedure
+. *Subscription ID*
+  - Login into your Microsoft Azure account.
+  - Select Subscriptions in the left sidebar.
+  - Select whichever subscription is going to be used.
+  - Click on overview.
+  - Copy the Subscription ID.
+. *Tenant ID*
+  - Login into your Microsoft Azure account.
+  - Select Microsoft Azure Active Directory in the left sidebar.
+  - Click properties.
+  - Copy the Microsoft Azure Active Directory ID.
+. *Client ID*
+  - Login into your Microsoft Azure account.
+  - Select Microsoft Azure Active Directory in the left sidebar.
+  - Click Enterprise applications.
+  - Click All applications.
+  - Select the application which you have created.
+  - Click Properties.
+  - Copy the Application ID.
+. *Client secret*
+  - Login into your Microsoft Azure account.
+  - Select Microsoft Azure Active Directory in the left sidebar.
+  - Click App registrations.
+  - Select the application which you have created.
+  - Click on All settings.
+  - Click on Keys.
+  - Type Key description and select the Duration.
+  - Click save.
+  - Copy and store the key value. You won't be able to retrieve it after you leave this page.
+
```

Comment:
I'm sorry to be a PITA, but we don't include the procedural steps for other products, especially not third party products in our own docs. 
Could we replace this with links to Microsoft Azure docs on the same?